sdk/python/kfp/components/_components.py

```diff
@@ -436,6 +438,15 @@ def expand_command_part(arg) -> Union[str, List[str], None]:
             else:
                 output_paths[output_name] = output_filename
 
+            # output_value_generator is only used by the v2 (experimental) compiler.
```

Comment:
I'm not sure this function is needed here. It looks like this work has to be performed by the component_bridge instead.

P.S. This file should not have a dependency on the dsl package or the compiler package.